docs/src/components/Layout/index.tsx

```diff
@@ -123,6 +123,7 @@ export default function Page({
                 >
                   <Icon
                     ariaLabel=""
+                    aria-hidden="true"
```

Comment:
This icon doesn't provide any context since the label already says "React". Axe complains because the SVG has a role of img and an empty title tag.